client/scss/components/_slider.scss

```diff
@@ -55,25 +57,37 @@ $slider-control-margin: 18px;
   }
 }
 
-.slider__slides-container { // these width and left calculations are related to the slider appearing in the article template
+.slider__slides-container {
   margin: 0;
   position: relative;
   width: calc(100vw - #{map-get($container-padding, 'small')});
 
   @include respond-to('medium') {
-    width: calc(100vw - #{map-get($container-padding, 'medium')} - 12.5%);
-    margin-left: 12.5%;
+    width: calc(100vw - #{map-get($container-padding, 'medium')});
+
+    &.slider__slides-container--in-article {
+      width: calc(100vw - #{map-get($container-padding, 'medium')} - 12.5%);
+      margin-left: 12.5%;
+    }
   }
 
   @include respond-to('large') {
     width: calc(100vw - #{map-get($container-padding, 'large')});
-    margin-left: 0;
+
+    &.slider__slides-container--in-article {
+      margin-left: 0;
+    }
   }
 
   @include respond-to('xlarge') {
-    width: $container-width-max - (map-get($container-padding, 'large') * 2);
-    margin-left: 50%;
-    left: (($container-width-max - (map-get($container-padding, 'large') * 2)) / 2) * (-5/6);
+    width: calc(100vw - ((100vw - #{$container-width-max}) / 2) - #{map-get($container-padding, 'large')});
+    left: calc((100vw - #{$container-width-max}) / 2);
+
+    &.slider__slides-container--in-article {
+      width: $container-width-max - (map-get($container-padding, 'large') * 2);
+      margin-left: 50%;
+      left: (($container-width-max - (map-get($container-padding, 'large') * 2)) / 2) * (-5/6);
+    }
```

Comment:
Let's have a chat with @Norvard about all the different ways we seem to do this.
Hopefully we can find one generic class to do this as we seem to do it in loads of places.